src/components/Dashboard/DashboardLayout.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavBar from '../navBar';
+import Searchbar from './Searchbar';
+
+function DashboardLayout() {
```

Comment:
I think we don't need a layout here, layout is defined if it going to be used on multiple pages

Reply:
We will be needing it in the future. as it will also have a table component.

Reply:
done